src/web/price-check/trade/common.ts

```diff
@@ -50,7 +50,8 @@ export function tradeTag (item: ParsedItem): string | undefined {
 
 const ENDPOINT_BY_LANG = {
   en: 'www.pathofexile.com',
-  ru: 'ru.pathofexile.com'
+  ru: 'ru.pathofexile.com',
+  zh_TW: 'www.pathofexile.com'
```

Comment:
Why don't you use poe.game.qq.com ?

Reply:
sry abort that, it's still testing, wrong click.